game_of_agents.py

```diff
@@ -50,20 +52,25 @@ def inrow_heuristic(env, player, weight=0):
 
                     # calculate the score of both directions
                     if counter_p * counter_n > 0:
-                        score += (counter_p + counter_n)**5
+                        score += score_func(counter_p + counter_n, weight)
                     else:
-                        score += (counter_p)**5 + (counter_n)**5
+                        score += score_func(counter_p, weight) + \
+                            score_func(counter_n, weight)
+    return score
+
+
+def score_func(count, weight):
+    score = count**POWER
```

Comment:
This function doesn't return anything, did you test your code by running the tournament script?

Reply:
Also, weight is not used in `score_func` - why are you passing it then?

Reply:
Did you see the all code of this function before you requested changes?

Reply:
Oh sorry, I didn't notice the lines below belonged to this function.